boards/arm/nrf9131ek_nrf9131/nrf9131ek_nrf9131_common.dtsi

```diff
@@ -156,6 +156,9 @@ arduino_spi: &spi3 {
 		reg = <0>;
 		spi-max-frequency = <8000000>;
 		size = <268435456>;
+		has-dpd;
+		t-enter-dpd = <3000>;
+		t-exit-dpd = <40000>;
```

Comment:
This should be moved to the first commit ("boards: arm: add nrf9131ek_nrf9131").